Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/person/core/MigrationCheckTest.kt

```diff
@@ -0,0 +1,89 @@
+package de.rki.coronawarnapp.covidcertificate.person.core
+
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CertificatePersonIdentifier
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CwaCovidCertificate
+import de.rki.coronawarnapp.main.CWASettings
+import io.kotest.matchers.shouldBe
+import io.mockk.MockKAnnotations
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.just
+import io.mockk.mockk
+import org.junit.jupiter.api.BeforeEach
+
+import org.junit.jupiter.api.Test
+
+internal class MigrationCheckTest {
+
+    @MockK private lateinit var cwaSettings: CWASettings
+
+    @BeforeEach
+    fun setup() {
+        MockKAnnotations.init(this)
+
+        every { cwaSettings.wasCertificateGroupingMigrationAcknowledged } returns false
+        every { cwaSettings.wasCertificateGroupingMigrationAcknowledged = any() } just Runs
+    }
+
+    @Test
+    fun `don't show migration info when no certificate is available`() {
+        val persons = emptySet<PersonCertificates>()
+
+        getInstance().shouldShowMigrationInfo(persons) shouldBe false
+    }
+
+    @Test
+    fun `don't show migration info when no change happens`() {
```

Comment:
Maybe also verify if `cwaSettings.wasCertificateGroupingMigrationAcknowledged = true` was called for this and the next test?

Reply:
ok, it will be updated in the another PR